andela/fargo-ah
# #159206049 Send verification mail to unverified user

#### What does this PR do?
- Send verification email to a newly created user or an unverified user
#### Description of Task to be completed?
- write units test for the response on sending, verifying and reverifying a user
- send verification email to new user
- verify generated token
- update user status to verified
#### How should this be manually tested?

POST /api/users

GET /api/users/verify

GET /api/users/recover

In postman navigate to `http://localhost:3000/api/users`
  
#### Any background context you want to provide?
- Pull in changes and run `npm install`
- Start server with `npm run dev`

#### What are the relevant pivotal tracker stories?
#159206049


[Delivers #159206049]
